donate.html

```diff
@@ -970,7 +970,7 @@ <h4>Shipping Address</h4>
         new expiration date, or new amount please visit <a href="https://350.org/update">350.org/update</a></p>
 
       <div style="float:left;margin:0 1em 1em 0;"><img
-          src="https://s3.amazonaws.com/s3.350.org/ak/charity_navigator_with_stars.png"
+          src="https://s3.amazonaws.com/s3.350.org/ak/charity_navigator_with_stars-min.png"

```

Comment:
Updated!

Reply:
The URL provided by AWS at `https://dbqvwi2zcv14h.cloudfront.net` is optimized via a [Cloudfront](https://aws.amazon.com/cloudfront/) service. Have you considered passing these resources via that CDN instead of directly accessing them?

Reply:
@mrkimani-ndegwa Sure that makes sense! I actually just used the existing link that we were using, and uploaded a smaller image to the same location. Do you think we should update this link to use Cloudfront instead of S3 as it was before?